x-pack/plugins/apm/public/components/app/RumDashboard/Charts/PageLoadDistChart.tsx

```diff
@@ -110,7 +111,7 @@ export function PageLoadDistChart({
             id="left"
             title={I18LABELS.percPageLoaded}
             position={Position.Left}
-            tickFormat={(d) => Number(d).toFixed(1) + ' %'}
+            tickFormat={(d) => numeral(d).format('0.0') + '%'}
```

Comment:
We should be using [`asPercent` from public/utils/formatters](https://github.com/elastic/kibana/blob/4845bef18194bf90778f6c156b7548dfc98f86c8/x-pack/plugins/apm/public/utils/formatters/formatters.ts#L27-L46)

Reply:
actually value is already in percent, this just adds a % postfix.

Reply:
This is going to make inconsistent legends between charts. If a value is a percent it should be passed around as 0.58039 or whatever and then formatted with `asPercent` when it's displayed.

Reply:
i see your point, so es `percentile_ranks` aggs returns values in %, we will have to divide that by 100 to return values as 0.58039. That just adds an extra step.